tests/config/dapr_postgres_query_state.yaml

```diff
@@ -16,12 +16,14 @@ kind: Component
 metadata:
```

Comment:
Why we still need a query state? Seems we have two postgres backend.

Reply:
This is what we had before, and these component names are referenced in tests. I do not want to extend this PR further by also updating the test code to change the component names used.